docs/HOWTO-configure-auth0.md

```diff
@@ -4,18 +4,17 @@ There are two authentication backends supported for Spoke.  The first and defaul
 [https://auth0.com](https://auth0.com). This service allows support for e.g. Google authentication
 and others, and supports things like password resets and account management separate from Spoke.
 
-The alternative and default for development environments is local login, where passwords are hashed locally
-in the database and resets, etc are administered all within Spoke.  While good for development, we
-believe Auth0 still provides better security for production environments.  Below are the steps to configure
-Spoke for Auth0.
+The alternative and default for development environments is local login, where passwords are hashed locally in the database and resets, etc are administered all within Spoke.  While good for development, we
+believe Auth0 still provides better security for production environments.  Below are the steps to configure Spoke for Auth0.
 
 ## Configuration Steps
 
-1. First configure the environment variable `PASSPORT_STRATEGY=auth0` in `.env` or wherever to configure Spoke environment
-variables.
+Note for users following [Instructions for One-Click Deployment to Heroku](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_HEROKU_DEPLOY.md): The fields to edit are listed alphabetically under the "Config Vars" heading.
+
+1. First configure the environment variable `PASSPORT_STRATEGY=auth0` in `.env` or wherever you configure Spoke environment variables.
 2. Create an [Auth0](https://auth0.com) account. In your Auth0 account, go to [Applications](https://manage.auth0.com/#/applications/), click on `Default App` and then grab your Client ID, Client Secret, and your Auth0 domain (should look like xxx.auth0.com). Add those inside your `.env` file (AUTH0_CLIENT_ID, AUTH0_CLIENT_SECRET, AUTH0_DOMAIN respectively).
 3. Run `yarn dev` to create and populate the tables.
-4. In your Auth0 app settings, set the following (Note: for development use `http://localhost:3000` instead of `https://yourspoke.example.com`):
+4. In your Auth0 app settings, set the following (Note: for development use `http://localhost:3000` instead of `https://yourspoke.example.com`. If following [Instructions for One-Click Deployment to Heroku](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_HEROKU_DEPLOY.md), use `https://<YOUR SPOKE APP>.herokuapp.com`.):
```

Comment:
How about:
```
4. In your Auth0 app settings, set the following (Note: for development use `http://localhost:3000` instead of `https://yourspoke.example.com`. If following [Instructions for One-Click Deployment to Heroku](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_HEROKU_DEPLOY.md), use `https://<YOUR SPOKE APP>.herokuapp.com` -- or if you have a vanity domain, the vanity domain which points to YOUR_SPOKE_APP.herokuapp.com as a CNAME in your DNS config.):
```